docs/install/install.rst

```diff
@@ -0,0 +1,587 @@
+.. _installing-flocker-node:
+
+Installing ``clusterhq-flocker-node``
```

Comment:
Change this to a non-monospaced phrase like "Installing the Flocker Node Services" for consistency with the client page title.